### PR TITLE
Set reference commit for migration checks

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]

**Review by acrellin — APPROVED** [review]

**Comment by stefanv** [comment]
Woohoo, thanks @acrellin!

**Comment by acrellin** [comment]
@stefanv What does this PR need now to be ready?

**Comment by stefanv** [comment]
I think it's good to go.

File: .github/workflows/test_migrations.yaml [hunk 1]
```diff
@@ -3,16 +3,21 @@ name: Test Skyportal Migrations
 on:
   # Run only if potential changes in database schema
   pull_request:
-    paths:
```
Comment (stefanv) [inline .github/workflows/test_migrations.yaml:6]: @acrellin One question, I guess, is whether this should go back in?

Comment (acrellin) [reply .github/workflows/test_migrations.yaml:6]: Ah, yeah probably good to restrict the runs to changes to these. Shall I add that back in a new commit?

Comment (stefanv) [reply .github/workflows/test_migrations.yaml:6]: Thanks, that'd be great; perhaps also set the reference commit to the point where the new data loader reporting got merged in, then we'll have the advantage of that when running migration tests.

**Pull request merged**